### PR TITLE
Fix: chip design

### PR DESCRIPTION
**Pull request by imrishabh18**

### CODE REVIEW

**Comment by imrishabh18** [comment]
@seveibar The chip still needs more work regarding the port spacing for the pins. Will create an issue for that and work

File: tests/examples/example4-kicad.test.tsx [hunk 1]
```diff
@@ -46,6 +47,11 @@ it("example 4: kicad theme demo", async () => {
       />
     </board>,
   )
+  const fs = require('node:fs');
+  const circuitJson = project.getCircuitJson();
+  fs.writeFileSync('circuit.json', JSON.stringify(circuitJson, null, 2)); 
 
   project.render()
+
+//   expect(project).toMatchSchematicSnapshot(import.meta.path)
```
Comment (seveibar) [inline tests/examples/example4-kicad.test.tsx:56]: Curious why we dont keep the snapshot

**Review by seveibar — APPROVED** [review]

File: tests/examples/example4-kicad.test.tsx (hunk 1 above)
Comment (imrishabh18) [reply tests/examples/example4-kicad.test.tsx:56]: Cause it's a circle, the new `circuit-to-svg` needs to be published for this snapshot to work

Comment (imrishabh18) [reply tests/examples/example4-kicad.test.tsx:56]: And for that to work, I need to publish this `core` 😆 

@seveibar

**Pull request merged**